powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/configuration/PowerAuthServiceConfiguration.java

```diff
@@ -155,6 +155,13 @@ public class PowerAuthServiceConfiguration {
     @Max(8)
     private int offlineSignatureComponentLength;
 
+    /**
+     * Expiration of timestamps for ECIES and MAC token requests.
+     */
+    @Value("${powerauth.service.crypto.requestExpirationInMilliseconds}")
+    @Min(0)
+    private int requestExpirationInMilliseconds;
```

Comment:
Just an idea, may we use `java.time.Duration` instead?

Reply:
Let's raise a separate issue, because we will need to refactor other usages as well.

Reply:
Ok, I have created #940